### PR TITLE
New Feature Created

### PR DESCRIPTION
**Pull request by Semiroundpizza8**
This adds a new feature in accordance to FEAT-41234

# Steps to test
- Open application
- Open README.md
- Make sure new feature change exists

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# SalesforceDayTwo
+# Salesforce Day Two
 
-Welcome to your second day of Git learning!
\ No newline at end of file
+Welcome to OUR second morning of Gittastic learning!
```
Comment (Semiroundpizza8) [inline README.md]: This line is a good change!

File: newFeature.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# New Feature Created Here!
```
Comment (Semiroundpizza8) [inline newFeature.md:1]: Do we really need this?

File: test.md [hunk 3]
```diff
@@ -1,3 +1 @@
 # This is a test!
-
-Yay!
```
Comment (Semiroundpizza8) [inline test.md:3]: Please add this back!

**Review by Semiroundpizza8 — COMMENTED** [review]
This is great, please make changes above!

File: README.md (hunk 1 above)
Comment (Semiroundpizza8) [reply README.md]: Here's how I would've done this feature / an example of what I'm thinking

```javascript
console.log("Hello World");
for(var i of array) {
  console.log(i)
}
```

Comment (atmartinezsf) [reply README.md]: test two
```suggestion
Welcome to OUR 2nd morning of Gittastic learning!
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,3 @@
-# SalesforceDayTwo
+# Salesforce Day Two
```
Comment (Semiroundpizza8) [inline README.md:1]: This can look like this:
```suggestion
asdfasdfasdf
```

**Pull request merged**